src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/FSharp/FSharpTypeConverterFactory.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    [RequiresUnreferencedCode(FSharpCoreReflectionProxy.FSharpCoreUnreferencedCodeMessage)]
```

Comment:
The feature that supports `RequiresUnreferencedCode` on classes is not implemented yet.

Instead of putting the attribute here, can you move it to an explicit, parameterless ctor?

[https://github.com/mono/linker/issues/1742#issuecomment-875036480](https://github.com/mono/linker/issues/1742#issuecomment-875036480)